flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/streaming/connectors/elasticsearch/table/ElasticsearchDynamicSink.java

```diff
@@ -45,22 +48,44 @@
  * logical description.
  */
 @Internal
-final class Elasticsearch7DynamicSink implements DynamicTableSink {
+class ElasticsearchDynamicSink implements DynamicTableSink {
```

Comment:
Does this mean we get rid of version specific ESDynamicSink in table? :D

Reply:
Yes :)